src/server/browser.ts

```diff
@@ -60,7 +61,7 @@ export abstract class Browser extends SdkObject {
   private _downloads = new Map<string, Download>();
   _defaultContext: BrowserContext | null = null;
   private _startedClosing = false;
-  readonly _idToVideo = new Map<string, Video>();
+  readonly _idToVideo = new Map<string, { context: BrowserContext, artifact: Artifact }>();
```

Comment:
I'm a little worried you just leaked all the contexts.

Reply:
Sure, let's clear these videos in `BrowserContext._didCloseInternal` just in case.